### PR TITLE
Modified the service-cluster mapping for regular service Znodes to be…

### PR DESCRIPTION
**Pull request by JieyingXu**
Modified the service-cluster mapping for regular service Znodes to be consistent with the change in d2-config-service.

Before the change, regular service Znode will be mapped to clusterA-localColo, e.g. clusterA-WestCoast.
After the change, it will be mapped to clusterA. The local colo suffix will not present.
This is to remove the requirements on operation orders when enabling/disabling cross colo routing to prevent errors.

### CODE REVIEW

**Review by rickzx — CHANGES_REQUESTED** [review]
Please make sure to update gradle.properties and run ./script/update_changelog

**Review by bohhyang — COMMENTED** [review]
echo Rick's comment.

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.26.4] - 2022-01-24
+- Modified the service-cluster mapping for regular service Znodes to be consistent with the change in d2-config-service.
```
Comment (rickzx) [inline CHANGELOG.md:18]: Let's not mention d2-config-service as this is a LI-internal MP.
I would prefer `Map local variant of service ZNodes to cluster without colo suffix`

**Review by rickzx — CHANGES_REQUESTED** [review]

**Review by rickzx — APPROVED** [review]

**Review by bohhyang — APPROVED** [review]
lgtm. thx

**Pull request merged**